### PR TITLE
Update dependencies and linter happiness

### PR DESCRIPTION
**Pull request by mkitt**
These guys changed:
screen shot 2016-10-12 at 3 38 17 pm

### CODE REVIEW

File: src/components/buttons/Buttons.css [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 .MiniPillButton {
   height: 30px;
-  padding: 0px 10px;
+  padding: 0 10px;
```
Comment (rynbyjn) [inline src/components/buttons/Buttons.css]: Cool, I like this update.

File: src/components/users/UserRenderables.js [hunk 2]
```diff
@@ -138,67 +138,67 @@ UserProfileCard.propTypes = {
...
+        onClickCollab={user.isCollaborateable ? onClickCollab : null}
+        onClickHireMe={user.isHireable ? onClickHireMe : null}
+      >
+        <RelationshipContainer
+          className="isPill inUserProfile"
+          relationshipPriority={user.relationshipPriority}
+          user={user}
+        />
+      </UserProfileButtons>
+    </div>
```
Comment (rynbyjn) [inline src/components/users/UserRenderables.js]: This looks just like indentation changes, right?

File: src/components/forms/RegistrationRequestForm.js [hunk 3]
```diff
@@ -96,7 +96,7 @@ class RegistrationRequestForm extends Component {
           />
           {(showEmailError && emailState.status !== STATUS.INDETERMINATE) ?
             <p className="HoppyStatusMessage hasContent">{emailState.message}</p> :
-            <p className="HoppyStatusMessage"><span /></p>
+              <p className="HoppyStatusMessage"><span /></p>
```
Comment (rynbyjn) [inline src/components/forms/RegistrationRequestForm.js]: Mmmm, beer 🍺

File: src/components/users/UserRenderables.js (hunk 2 above)
Comment (mkitt) [inline src/components/users/UserRenderables.js]: yep

**Comment by mkitt** [comment]
Bummer that this didn't fix node security. 👎

**Pull request merged**